### ISSUE TITLE
BUG - 2 - All - Accessibility: User cannot swipe left/right out of Date Ranger picker after selecting option

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#12102]
## What happened?

When an accessibility user is on a page with a Date Range picker (such as Travel Claims), after the user selects an option from the date range pop-up menu, the user cannot swipe left/right to navigate out of the menu to focus on other page elements (such as the date range results).

## Specs:

- Device: Android/iOS
- OS: Latest
- User Account: DEMO
- Accessibility State: Screen reader ON, default settings.

## Steps to Reproduce

1. Log into the app as the Demo user.
2. Go to the Benefits > Claims > Travel Claims page.
3. Open the Date Picker and select any option from the pop-up menu.
4. After the results from the date picker load, attempt to swipe left/right to navigate focus off of the date range menu.
5. Observe - Swiping left/right does not exit the user from the focus of the Date Range selector. The user must use a different gesture navigation to exit from the menu focus. (Such as swiping two fingers up).

## Acceptance Criteria

After a user has made a selection from the Date Picker menu (and selected the 'Done' button), the user should be able to easily navigate off of the Date Range menu.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High **Low** 
- Frequency: **High** Low 

- 3 - Low
- 2 - **High**
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by digitalken** [comment]
Pass.  User can swipe left or right to exit date picker after choosing a value. OK on iOS and Android.
